src/gesture-recognizers.js

```diff
@@ -86,7 +90,11 @@ export class TapRecognizer extends GestureRecognizer {
 
   /** @override */
   acceptStart() {
-    this.signalEmit({clientX: this.lastX_, clientY: this.lastY_}, null);
+    this.signalEmit({
+      clientX: this.lastX_,
+      clientY: this.lastY_,
+      target: this.target_,
```

Comment:
The only thing I added here is the `target_` variable.

Reply:
let's modify one of the tests in test-gesture-recognizer.js to make sure target is set.